manage-data/data-store/data-streams/time-series-data-stream-tsds.md

```diff
@@ -78,146 +61,68 @@ You mark a field as a dimension using the boolean `time_series_dimension` mappin
+#### `_tsid` metadata field [tsid]
 
-[Pass-through](elasticsearch://reference/elasticsearch/mapping-reference/passthrough.md#passthrough-dimensions) fields may be configured as dimension containers. In this case, their sub-fields get included to the routing path automatically.
+The `_tsid` is an automatically generated object containing the document’s dimensions. It's intended for internal {{es}} use, so in most cases you won't need to work with it. The format of the `_tsid` field is subject to change. 
 
-TSDS documents don’t support a custom `_routing` value. Similarly, you can’t require a `_routing` value in mappings for a TSDS.
+### Differences from a regular data stream [differences-from-regular-data-stream]
 
+A time series data stream works like a regular data stream, with some key differences:
 
-### Index sorting [tsds-index-sorting]
+* **Time series index mode:** The matching index template for a TSDS must include a `data_stream` object with `index.mode` set to `time_series`. This option enables most TSDS-related functionality.
+* **Fields:** In a TSDS, each document contains:
+  * A `@timestamp` field
+  * One or more [dimension fields](#time-series-dimension), set with `time_series_dimension: true`  
+  * One or more [metric fields](#time-series-metric)
+  * An auto-generated document `_id` (custom `_id` values are not supported)
+* **Backing indices:** A TSDS uses [time-bound indices](/manage-data/data-store/data-streams/time-bound-tsds.md) to store data from the same time period in the same backing index.
+* **Dimension-based routing:** The routing logic uses dimension fields to map data to shards, improving storage efficiency and query performance.
+* **Sorting:** A TSDS uses internal [index sorting](elasticsearch://reference/elasticsearch/index-settings/sorting.md) to order shard segments by `_tsid` and `@timestamp`, for better compression. Time series data streams do not use `index.sort.*` settings.
 
-{{es}} uses [compression algorithms](elasticsearch://reference/elasticsearch/index-settings/index-modules.md#index-codec) to compress repeated values. This compression works best when repeated values are stored near each other — in the same index, on the same shard, and side-by-side in the same shard segment.
+## Query time series data
+```{applies_to}
+stack: preview
+serverless: preview
+```
 
-Most time series data contains repeated values. Dimensions are repeated across documents in the same time series. The metric values of a time series may also change slowly over time.
+You can use the {{esql}} [`TS` command](elasticsearch://reference/query-languages/esql/commands/ts.md) (in technical preview) to query time series data streams. The `TS` command is optimized for time series data. It also enables the use of aggregation functions that efficiently process metrics per time series, before aggregating results.
```

Comment:
Maybe remove the (in technical preview). The TS docs also contain that and it's another thing we need to remember to keep in sync when TS goes beta or GA.